less/components/teach-like-mozilla-page.less

```diff
@@ -0,0 +1,32 @@
+.teach-like-mozilla {
+  h1, h2 {
+    margin: 0;
+    padding: 0;
+  }
+  .hero-unit h2 {
+    font-size: 3rem;
+  }
+  .hero-unit h1 {
+    max-width: 100%;
+    padding: 0.8rem 0px;
+  }
+  .sub-title {
+    font-style: italic;
+    font-size: 2.8rem;
+  }
+  .list-with-illust {
+    margin-top: 4rem;
+    img {
+      width: 100%;
+      max-width: 240px;
+      margin: 4rem auto 4rem;
+    }
+    ul li {
```

Comment:
There's a mixin for this—`.listItemBorderColorizer(@borderColor:@bodyTextColor)`

(also using `ul li` is more complex than necessary)